templates/zerver/for/rust-case-study.md

```diff
@@ -0,0 +1,167 @@
+Wood](https://davidtw.co/), member of the [Rust compiler
+team](https://www.rust-lang.org/governance/teams/compiler). “In Zulip, I can
+instantly see the context for each message.”
+
+Afterwards, Zulip’s thread-based organization creates a **clear record** of
+past discussions, documenting the decision-making process. “Zulip creates a
+transparent record of what we do,” says Josh. It’s **easy to link** to Zulip
+threads from other tools in order to provide background context. “We link to
+Zulip threads all the time on GitHub, Twitter, email, Discourse, and from other
+Zulip messages,” [Josh Triplett](https://joshtriplett.org/) says.
+
+
+> “Slack and Discord feel opaque. Zulip feels like an open room.”
+>
+> — [Rust Language team](https://www.rust-lang.org/governance/teams/lang) co-lead
+> [Josh Triplett](https://joshtriplett.org/)
+
+Because information on Zulip is **well-organized**, one can easily find and
+review earlier conversations, even if months have passed. The organization’s
+**rich history** continues to provide value indefinitely.  “I can find and
+re-read old conversations,” says [Rust Language
+team](https://www.rust-lang.org/governance/teams/lang) co-lead [Niko
+Matsakis](https://github.com/nikomatsakis). “If I send someone a link to a
+discussion, they can jump right in; this would be impossible in other chat tools
+I’ve used.”
+
+Unlike other chat tools, a discussion on Zulip can even be revived after weeks
+or months have passed if new information or ideas come to light, with no loss of
+**surrounding context**.
+
+Catching up works great, whether you were away for hours, days, or even weeks.
+“When I come back after a break, I don’t feel overwhelmed: I can skim topics
+looking for the ones that seem important, and scroll briefly through the rest,”
+says [Rust Language team](https://www.rust-lang.org/governance/teams/lang)
+co-lead [Niko Matsakis](https://github.com/nikomatsakis).
+
+
+> “When I come back after a break, I don’t feel overwhelmed.”
+>
+> — [Rust Language team](https://www.rust-lang.org/governance/teams/lang)
+> co-lead [Niko Matsakis](https://github.com/nikomatsakis)
+
+
+## Zulip makes the Rust community more agile
+
+In addition to being well-organized, Zulip interactions are
+**conversational**, enabling **faster decision-making**. GitHub issues about
+controversial Rust language decisions can devolve into incomprehensible comment
+threads with [several](https://github.com/rust-lang/rust/issues/57640)
+[hundred](https://github.com/rust-lang/rust/issues/34511)
+[messages](https://github.com/rust-lang/rust/issues/28237).  “Some decisions
+that were blocked for months on GitHub were resolved within 24 hours by starting
+the right conversation on Zulip,” says [Rust Language
+team](https://www.rust-lang.org/governance/teams/lang) co-lead [Josh
+Triplett](https://joshtriplett.org/).
+
+
+> “Some decisions that were blocked for months on GitHub were resolved within 24
+> hours on Zulip.”
```

Comment:
I think a lot of the quotes in this file seem like they're using the same curly quote symbol for both opening and closing of quotations, and we should be using “quote”.

Reply:
Those look different to me... can you explain what you mean?